packages/helpers/src/components/demo.tsx

```diff
@@ -17,6 +18,8 @@ interface IProps<IAuthoredState, IInteractiveState> {
 const iframe = new URLSearchParams(window.location.search).get("iframe");
 const rootDemo = !iframe;
 
+const dynamicTextProxy = useDynamicTextProxy();
```

Comment:
I'm surprised the linter isn't complaining about something that looks like a hook being used outside of a function. 🤷

Reply:
Its a poorly named function.  It's not a hook but a getter to return a singleton of a class:

https://github.com/concord-consortium/dynamic-text/blob/main/src/proxy.ts#L44

Reply:
But the `rules-of-hooks` linter rule assumes that a function that starts with `use...` _is_ a hook, which is the puzzling part of @scytacki's question. My suspicion would be that the rules of hooks are only applied within React components. 🤷